docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -3492,6 +3492,158 @@ toFloat64OrDefault('abc', CAST('0', 'Float64')): 0
 - [`toFloat64OrZero`](#tofloat64orzero).
 - [`toFloat64OrNull`](#tofloat64ornull).
 
+## toBFloat16
+
+Converts an input value to a value of type [`BFloat16`](../data-types/float.md/#bfloat16). 
+Throws an exception in case of an error.
+
+**Syntax**
+
+```sql
+toBFloat16(expr)
+```
+
+**Arguments**
+
+- `expr` — Expression returning a number or a string representation of a number. [Expression](../syntax.md/#syntax-expressions).
+
+Supported arguments:
+- Values of type (U)Int8/16/32/64/128/256.
+- String representations of (U)Int8/16/32/128/256.
+- Values of type Float32/64, including `NaN` and `Inf`.
+- String representations of Float32/64, including `NaN` and `Inf` (case-insensitive).
+
+**Returned value**
+
+- 16-bit floating point data type with 8-bit exponent, sign, and 7-bit mantissa. [BFloat16](../data-types/float.md/#bfloat16).
```

Comment:
Seems too verbose, can we say here (\+ below) just "- 16-bit brain-float value. [BFloat16](../data-types/float.md/#bfloat16)`